### PR TITLE
PJ_SIM_SUNBNB_2023-9 [FE] Layout Component with Navbar

### PR DESCRIPTION
**Pull request by andreenriquez11**
# Issue Link
https://framgiaph.backlog.com/view/PJ_SIM_SUNBNB_2023-9

# Slack Link
https://sun-philippine.slack.com/archives/C05GURS7N3U/p1690342911993489

# Definition of Done
* [x] Create a markup for the layout component with the navigation bar with the components:
    * [x] Title
    * [x] Search bar
    * [x] Become a Host Button
    * [x] Profile Dropdown linking to Profile Page and Edit Profile(shown when logged In)
    * [x] Login Button (shown when not logged in)
    * [x] Sign Up Button (shown when not logged in)

# Notes
- Added navbar to App.vue to be visible in all pages
- Had set-up a global css for uniform font and margin for all pages
- Shown all the buttons and also the profile button for visual purposes, will add logic after all FE tasks are done

# Pre-conditions
- Profile button should only be visible when the user is logged in but I have shown it even when not logged in for visual and checking purposes only

# Expected Output
image

# Screenshots/Recordings
image
image

### CODE REVIEW

**Review by LennardUy104 — COMMENTED** [review]
Rebase lng so that ang changes sa review will not affect the branch 

Overall looks good and working, Nice!

**Review by roseaugusto — COMMENTED** [review]
You can actually group same components into one, so please group navbar into one folder and you can just put index.js as indicator of the main component 🙇🏼‍♀️

File: fe/src/components/ReviewHost.vue [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                    <p class="d-flex align-end">{{ review.review }}</p>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+    export default {
+        data: () => ({
```
Comment (roseaugusto) [inline fe/src/components/ReviewHost.vue]: as I instructed, please use dummy file for this one

**Review by andreenriquez11 — COMMENTED** [review]
@roseaugusto I have successfully grouped the same components into one folder of Navbar maam. Please review if you have time.

**Review by roseaugusto — APPROVED** [review]

**Pull request merged**